renderdoc/driver/gl/wrappers/gl_texture_funcs.cpp

```diff
@@ -2816,6 +2816,81 @@ void WrappedOpenGL::glCompressedMultiTexImage1DEXT(GLenum texunit, GLenum target
   }
 }
 
+void WrappedOpenGL::StoreCompressedTexData(ResourceId texId, GLenum target, GLint level,
+                                           GLint xoffset, GLint yoffset, GLint zoffset,
+                                           GLsizei width, GLsizei height, GLsizei depth,
+                                           GLenum format, GLsizei imageSize, const void *pixels)
+{
+  bool dataStored = false;
+  byte *unpackedPixels = NULL;
+  byte *srcPixels = NULL;
+  GLint unpackbuf = 0;
+
+  m_Real.glGetIntegerv(eGL_PIXEL_UNPACK_BUFFER_BINDING, &unpackbuf);
+
+  if(unpackbuf == 0 && pixels != NULL)
+  {
+    PixelUnpackState unpack;
+    unpack.Fetch(&m_Real, false);
+
+    if(unpack.FastPathCompressed(width, height, depth))
+      srcPixels = (byte *)pixels;
+    else
+      srcPixels = unpackedPixels =
+          unpack.UnpackCompressed((byte *)pixels, width, height, depth, imageSize);
+  }
+
+  if(unpackbuf != 0)
+    srcPixels = (byte *)m_Real.glMapBufferRange(eGL_PIXEL_UNPACK_BUFFER, (GLintptr)pixels,
+                                                imageSize, eGL_MAP_READ_BIT);
+
+  if(srcPixels)
+  {
+    // Only the trivial case is handled yet.
```

Comment:
Can you add a specific warning message for when we hit a case that isn't handled? Then it's also clear which cases still have to be implemented so that the warnings can be removed incrementally as each case is handled. E.g. one for xoffset/yoffset, one for zoffset in 3D case, one for multiple layers at once in the array case.